### PR TITLE
Fix issue with preparing Graphics without shader

### PR DESCRIPTION
**Pull request by YuryKuvetski**
_resolveDirectShader is needed in a renderer parameter, otherwise it will fail [there](https://github.com/pixijs/pixi.js/blob/dev/packages/graphics/src/Graphics.ts#L1157)

##### Description of change
Just passing down renderer to the _resolveDirectShader.

##### Pre-Merge Checklist

- [x] Lint process passed (`npm run lint`)
- [x] Tests passed (`npm run test`)

### CODE REVIEW

**Comment by ivanpopelyshev** [comment]
oops :)

File: packages/prepare/src/Prepare.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ function uploadGraphics(renderer: AbstractRenderer | BasePrepare, item: IDisplay
     // if its not batchable - update vao for particular shader
     if (!geometry.batchable)
     {
-        (renderer as Renderer).geometry.bind(geometry, (item as any)._resolveDirectShader());
+        (renderer as Renderer).geometry.bind(geometry, (item as any)._resolveDirectShader((renderer as Renderer)));
```
Comment (bigtimebuddy) [inline packages/prepare/src/Prepare.ts:70]: Why is `item` cast as any? We should have caught this issue way sooner.

Comment (YuryKuvetski) [reply packages/prepare/src/Prepare.ts:70]: Some kind of legacy?

Comment (bigtimebuddy) [reply packages/prepare/src/Prepare.ts:70]: I think was just an issue with with this package being converted to TypeScript #6481. We should followup with a PR, that create an Interface for batchable objects.

**Pull request merged**